### PR TITLE
Merge release 1.12 to staging

### PR DESCRIPTION
**Pull request by coutoPL**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: designer/client/src/components/spinner/Spinner.tsx [hunk 1]
```diff
@@ -12,11 +12,11 @@ function LoaderSpinner(props: Props) {
 export default LoaderSpinner;
 
 function CircularProgressWrapper() {
-    const theme = useTheme();
```
Comment (coutoPL) [inline designer/client/src/components/spinner/Spinner.tsx:15]: @Dzuming please verify this

Comment (Dzuming) [reply designer/client/src/components/spinner/Spinner.tsx:15]: this 

`const theme = useTheme();`

is the correct one

Comment (coutoPL) [reply designer/client/src/components/spinner/Spinner.tsx:15]: ok, fixed

**Review by Dzuming — APPROVED** [review]
LGTM

**Pull request merged**